pkgs/development/libraries/gdal/default.nix

```diff
@@ -88,6 +89,7 @@ stdenv.mkDerivation rec {
     "-DGEOTIFF_LIBRARY_RELEASE=${lib.getLib libgeotiff}/lib/libgeotiff${stdenv.hostPlatform.extensions.sharedLibrary}"
     "-DMYSQL_INCLUDE_DIR=${lib.getDev libmysqlclient}/include/mysql"
     "-DMYSQL_LIBRARY=${lib.getLib libmysqlclient}/lib/${lib.optionalString (libmysqlclient.pname != "mysql") "mysql/"}libmysqlclient${stdenv.hostPlatform.extensions.sharedLibrary}"
+    "-DBUILD_TESTING=ON"
```

Comment:
This should depend on doCheck.

Reply:
@SuperSandro2000 , do you mean doing something like

```
++ lib.optionals doCheck
```

?

Reply:
yep

Reply:
@kidanger , are you able to update your PR with this change ? Thanks.

Reply:
Done, thanks!

Reply:
Sorry, we should check `doInstallCheck` instead.

This 

```
  ] ++ lib.optionals doInstallCheck [
```

works.